.github/workflows/ci-workflow.yml

```diff
@@ -1,6 +1,12 @@
 name: CI Workflow
 
-on: [push, pull_request]
+on:
+  push:
+    paths-ignore:
+      - 'website/**'
+  pull_request:
+    paths-ignore:
+      - 'website/**'
```

Comment:
Can also include *.txt, *.md

Reply:
You mean txt and md files in the entire repo?